### DISCUSSION TITLE
Stop insta-closing discussions and issues before anyone can even respond

### DISCUSSION THREAD

**Discussion by TobiTenno** [Ash258/scoop#276] [Ideas]
Ash, I get that you're frustrated with how things went down with scoop and the lack of maintenance, but blowing off well-meaning users who are trying to support and use your fork is not the way to endear yourself to your consumers and people trying to get acquainted enough to also help and maintain buckets/apps.

**Comment by Ash258** [comment]
I do not need your support. You are not anyhow forced to use my fork. Use a vanilla scoop if you wish.

I am closing the issues when there is no need for discussion. I am not going to give you alternative. You are not allowed to add exclusion to shovel current folder? Do not use it.
End of story.
